pages/api/labs/questionGenerators/coinsQuestionsGenerator.ts

```diff
@@ -0,0 +1,77 @@
+import { Question } from "../../question";
+import { Skill } from "../../skill";
+import { QuestionType } from "../../questionTypes";
+import {
+    getRndInteger,
+    getRndTenthsDecimal,
+    getRndHundredthsDecimal,
+    getRandomItemFromArray,
+  } from "../../random";
+import { CoinType, Coin, CoinProp } from "../../../../components/questionTypes/finance/Coin";
+import { randomize } from "./binaryQuestionGenerator";
+
+
+
+export type CountingCoinsQuestion = {
+    questionType: QuestionType.FINANCE_COUNTING_COINS_BILLS_PROBLEM;
+    //coins: CoinType[];
+    answer: string;
+    countingModel: CountingModel;
+    text: string;
+   };
+
+
+export type CountingModel = {
+    coins: CoinType[];
+    };
+
+
+export function generateCountingQuestion() {
+    // this below code is for randomizing the coins that show, need that, also need to limit value. look at for loops. 
+    var coinsArray = [];
+    
+    var answerCoin = 0;
+    for(var i=0;i<6;i++){
+        var randomNumber = randomize(1,6);        
```

Comment:
I would rename this variable to randomCoinChoice instead of randomNumber since you are picking a coin from random